cfme/v2v/migration_plans.py

```diff
@@ -511,8 +511,13 @@ def create(
             'pre_checkbox': pre_checkbox,
             'post_checkbox': post_checkbox
         })
+
+        # Schedule migration check
         view.schedule.wait_displayed()
-        view.schedule.fill(start_migration)
+        start_selection = ["Start migration immediately"
```

Comment:
Required :
start_selection = "Start migration immediately" if start_migration else "Save migration plan to run later"